docs/Common-Issues.md

```diff
@@ -113,6 +113,90 @@ xcode-select --install
 brew upgrade
 ```
 
+## Cask - cURL error
+
+First, let's tackle a common problem: do you have a `.curlrc` file? Check with `ls -A ~ | grep .curlrc` (if you get a result, the file exists). Those are a frequent cause of issues of this nature. Before anything else, remove that file and try again. If it now works, do not open an issue. Incompatible `.curlrc` configurations must be fixed on your side.
+
+If, however, you do not have a `.curlrc` or removing it did not work, let’s see if the issue is upstream:
+
+1. Go to the vendor’s website (`brew home <cask_name>`).
+2. Find the download link for the app and click on it.
+
+### If the download works
+
+The cask is outdated. Let’s fix it:
+
```

Comment:
Could/should we mention something here about using `brew livecheck {{cask_name}}` to check if there is a new version, followed by running `brew bump-cask-pr`?

Reply:
I think all guides could use some love and updated instructions, but I'd prefer to do that in a later PR. This one is really about moving everything.